### PR TITLE
Add Parrable ID System Docs

### PR DESCRIPTION
**Pull request by icflournoy**
This is a merge into the branch that will be submitted upstream.

### CODE REVIEW

**Review by justinegreene — APPROVED** [review]
I'm ok with my changes.  I would like someone to read them and sanity check me please.

**Comment by eyas-ranjous** [comment]
will do on the afternoon.

**Review by eyas-ranjous — APPROVED** [review]

File: dev-docs/modules/userId.md [hunk 1]
```diff
@@ -25,6 +25,7 @@ The User ID module supports multiple ways of establishing pseudonymous IDs for u
 * **ID5 ID** - a neutral identifier for digital advertising that can be used by publishers, brands and ad tech platforms (SSPs, DSPs, DMPs, Data Providers, etc.) to eliminate the need for cookie matching.
 * **Criteo RTUS ID** – fetches a user id by reaching out to Criteo rtus endpoint for each bidder configured. The result is stored in the user's browser for 1 hour and is passed to bidder adapters to pass it through to SSPs and DSPs that support the ID scheme.
 * **Identity Link** – provided by LiveRamp, this module calls out to the ATS (Authenticated Traffic Solution) library or a URL to obtain the user’s IdentityLink envelope.
+* **Parrable ID** - an encrypted psudonymous id that is consistent across all browsers and webviews on a device for every publisher the device visits.  This module contacts Parrable to obtain the Parrable EID belonging to the specific device which can then be used by the bidder.
```
Comment (icflournoy) [inline dev-docs/modules/userId.md]: (sp) pseudonymous?
I am uncertain that this is the correct word here.

also capitalizing ID for consistency

File: dev-docs/modules/userId.md [hunk 2]
```diff
@@ -432,6 +433,50 @@ pbjs.setConfig({
...
+{: .table .table-bordered .table-striped }
+| Param under usersync.userIds[] | Scope | Type | Description | Example |
+| --- | --- | --- | --- | --- |
+| params | Required | Object | Details for the Parrable ID. | |
+[ params.partner | Required | Number | A list of one or more comma-separated Parrable Partner Client IDs for the Parrable-aware bid adapters you are using.  Please obtain Parrable Partner Client IDs from them and/or obtain your own. | `'30182847-e426-4ff9-b2b5-9ca1324ea09b'` |
+
+{: .alert.alert-info :}
+NOTE: The Parrable ID that is delivered to Prebid is encrypted by Parrable with a time-based key and updated frequently in the browser to enforce consumer privacy requirements and thus will be different on every page view, even for the same user.
+ 
+We recommend setting `storage.expires` to no more than`364` days, which is the default cookie expiration that Parrable uses in the standalone parrable integration.
```
Comment (icflournoy) [inline dev-docs/modules/userId.md]: capitalize Parrable

**Pull request merged**